packages/faency/src/components/Avatar.tsx

```diff
@@ -27,7 +26,7 @@ export const Avatar = forwardRef<HTMLSpanElement, AvatarProps>((props, forwarded
               color: '#fff',
               transition: 'all 200ms ease-in-out',
               '*': {
-                color: '#fff !important',
+                color: '#fff',
```

Comment:
I think we should keep the important here, it was important for this scenario:

![image](https://user-images.githubusercontent.com/38889364/78253311-9168c600-74ca-11ea-950d-3b924f7eefb2.png)

Then the text can stay white.

Reply:
For me this is the expected color displayed there. See it as it: "being in dark mode I want to display the color having as constant "white"". And in our dark theme the "white" constant has as value the black color. The issue could be the way we flip colors between light and dark mode but not what's happening there. Moreover we should try to forbid usage of `!important` in a ui lib.

Reply:
Yeah you’re right, I agree.